microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -1,103 +1,110 @@
 
-    ![Open Device Portal](./media/open-device-portal.png)
+1.  If you are only debugging apps that are running locally, you can select **Continue with an unsecured connection**.
 
-1. If you are only debugging apps running locally, you can select **Continue with an unsecured connection**.
-    Otherwise if you are not running the app locally, or if you need to connect over `HTTPS` select **Configure a secure connection** and follow the steps outlined in the page you are redirected to.
+    Or, if you aren't running the app locally, or if you need to connect over `HTTPS`, select **Configure a secure connection**, and then follow the steps in the page that you're redirected to.
 
-1. Verify the page you are redirected to is as follows:
+1.  Verify that the page you're redirected to is as follows:
 
-    ![Device Portal](./media/device-portal.png)
+    ![Device Portal](./remote-debugging-images/device-portal.png)
 
-1. Navigate to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page should load an empty list `[]`.
+1.  Go to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page should load an empty list: `[]`.
 
-1. Configure your WebView2 WinUI 2 (UWP) app for remote debugging:
+1.  Configure your WebView2 WinUI 2 (UWP) app for remote debugging:
 
-    To enable remote debugging, two environment variables must be set in your project.
-
-    These variables must be set before the `CoreWebView2` creation before either the `WebView2.Source` property is set or the `WebView2.EnsureCoreWebView2Async` method is called.
+    To enable remote debugging, two environment variables must be set in your project.  These variables must be set before creating the `CoreWebView2` instance, and before either setting the `WebView2.Source` property or calling the `WebView2.EnsureCoreWebView2Async` method.
 
     Set the following variables:
 
-    `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS = --enable-features=msEdgeDevToolsWdpRemoteDebugging`
+    ```
+    "WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging"
+    "WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1"
+    ```
+    
+    For example, if you're using the [WebView2 Sample UWP app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/webview2_sample_uwp), you can set those environment variables by adding the following lines in the `Browser.xaml.cs` file:
     
-    `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE = 1`
+    ```csharp
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging");
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1");
+    ```
+
+    Those are lines 33 and 34 in this screenshot:

```

Comment:
what is "Those"?, code settings?

Reply:
Changed to "the".  This is referring to the environment variables.